src/detect-engine.c

```diff
@@ -2306,6 +2311,111 @@ static void InjectPackets(ThreadVars **detect_tvs,
     }
 }
 
+/**
+ * \brief Trigger detect threads to flush their output logs
+ *
+ * This function is intended to be called at regular intervals to force
+ * buffered log data to be persisted
+ */
+void DetectEngineFlush(void)
```

Comment:
Suggest a different name, e.g., "WorkerFlush", because it's not really part of the "DetectEngine"